Frontend/index.html

```diff
@@ -54,27 +57,27 @@
         <h1>O projekte</h1>
         <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corrupti illo ratione aut in quaerat saepe minima atque ad, voluptatum non assumenda quisquam modi tempore numquam reprehenderit asperiores mollitia neque! Ex. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Excepturi unde ad explicabo quod adipisci. Labore necessitatibus aut nihil debitis magnam maiores, sunt, sit odit est ea, beatae mollitia. Voluptatum, exercitationem.</p>
     </div>
-    <div style="line-height: 50px; font-size: 23px; background-color: rgb(145,140,140, 0.3);; color: #302d2b; text-align: center;">Na projekte pracovali:</div>
+    <div class="napr_prac">Na projekte pracovali:</div>
     <div class="about_us">
         <div class="column"><!--<img src="assets/img/12.jpg" alt="1">-->
            <h3 style="text-align: center;
            padding:  20px 0;">Maros</h3>
-            <p class="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nihil maiores accusamus temporibus, quas beatae, voluptates quod sed ratione quibusdam, iusto velit a. Distinctio optio quibusdam qui nisi autem, eos voluptate.</p>
+            <p class="text">Pracoval som na koncepte tohto projektu a hardware pre Node jednotky. Mám 19 rokov a som absolvent Gymnázia Jána Adama Raymana.</p>
         </div>
         <div class="column"><!--<img src="assets/img/12.jpg" alt="2">-->
            <h3 style="text-align: center;
            padding:  20px 0;">Samo</h3>
-            <p class="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nihil maiores accusamus temporibus, quas beatae, voluptates quod sed ratione quibusdam, iusto velit a. Distinctio optio quibusdam qui nisi autem, eos voluptate.</p>
+            <p class="text">Pracoval som na softvéri a správe servera. Mám 20 rokov a som absolventom Gymnázia Jána Adama Raymana.</p>
         </div>
         <div class="column"><!--<img src="assets/img/12.jpg" alt="3">-->
            <h3 style="text-align: center;
            padding:  20px 0;">Daniel</h3>
-            <p class="text">Pracoval som na dizajne stranky. Mam 16 rokov. Navstevujem gymnazium Jana Adama Raymana v presove a som ziakom 2.D triedy. </p>
+            <p class="text">Pracoval som na dizajne stránky. Mám 16 rokov. Navštevujem Gymnázium Jána Adama Raymana v Prešove a som žiakom 2.D triedy. </p>
         </div>
         <div class="column"><!--<img src="assets/img/12.jpg" alt="4">-->
            <h3 style="text-align: center;
            padding:  20px 0;">Miro</h3>
-            <p class="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nihil maiores accusamus temporibus, quas beatae, voluptates quod sed ratione quibusdam, iusto velit a. Distinctio optio quibusdam qui nisi autem, eos voluptate.</p>
+            <p class="text">Pracoval som na softvéri pre Node. Mám 18 rokov. Navštevujem gymnázium Jána Adama Raymana v Prešove a som žiakom 4.C triedy.</p>

```

Comment:
```suggestion
            <p class="text">Pracoval som na softvéri pre Node. Mám 18 rokov. Navštevujem Gymnázium Jána Adama Raymana v Prešove a som žiakom 4.C triedy.</p>
```
Ešte tu jedno, nevšimol som si to predtým.

Reply:
@AVATAR197